onnx2tf/onnx2tf.py

```diff
@@ -713,8 +715,7 @@ def convert(
                 model_input.name for model_input in model.inputs
             ]
             data_count = 0
-            if output_integer_quantized_tflite \
-                and quant_calib_input_op_name_np_data_path is None \
```

Comment:
Reject this modification because it is necessary to generate a Float32 model, as per the comment in Line.670.

Reply:
I don't understand the objection. The check on line 692 tests if output_integer_quantized_tflite is true. In the nested ifs on lines 716 and 742 you the code checks output_integer_quantized_tflite again, which is known to be true. Why check it again?

Reply:
Your suggested modification is correct.